### PR TITLE
Hotfix remove staff restriction in toolbar for user management link

### PR DESCRIPTION
**Pull request by lemniscate613**
Backport hotfix to QC.

**NOTE**: This is my first backport of a hotfix. I think I followed @tdphillips's pattern ( https://github.com/DirectEmployers/MyJobs/pull/2071 ). I simply took my original hotfix branch to master ( https://github.com/DirectEmployers/MyJobs/pull/2073 ), pulled the latest changes to master (there have been a few hotfixes and merges since I created it), and created this PR to merge into QC. Therefore there are a few more commits listed below than are directly relevant to my original hotfix.

### CODE REVIEW

**Comment by aspidites** [comment]
Need to update with QC.

**Comment by lemniscate613** [comment]
Thanks, one second.

File: registration/tests/test_views.py [hunk 1]
```diff
@@ -230,6 +230,30 @@ def test_permission_invitation_message(self):
         self.assertIn("in order to help administer their recruitment and "
                       "outreach tools.", email.body)
 
+    def test_invite_new_user_shows_password(self):
```
Comment (aspidites) [inline registration/tests/test_views.py]: This is wrong. I'm having the same failure in my branch. You'll either need to cherry pick the changes when I'm done do the same fix. In other words -- you won't pass tests as is.

Comment (lemniscate613) [inline registration/tests/test_views.py]: Okay, if you let me know when you're done I'll update this branch/PR.

Comment (aspidites) [inline registration/tests/test_views.py]: While this is adequate for master, the correct fix (which this test relies upon) will require a more significant refactor. 

I'll leave it to @JLMcLaugh  whether he wants to merge this without that test passing sense it's related to other functionality and we have established a precedent for merging things regardless of test status.

**Pull request merged**